contracts/src/NobleDollar.sol

```diff
@@ -84,9 +84,7 @@ contract NobleDollar is HypERC20 {
         }
     }
 
-    constructor(address mailbox_) HypERC20(6, 1, mailbox_) {
-        _disableInitializers();
-    }
+    constructor(address mailbox_) HypERC20(6, 1, mailbox_) {}
```

Comment:
Would be great to know why you removed this, given that we will be deploying this contract as well (just not on Hyperliquid)